### PR TITLE
Release develop to master

### PR DESCRIPTION
**Pull request by snario**
### This is not how `master` will be updated, it just shows the diff.
`master` will be rebased on `develop` and force pushed.

The following packages are currently deployed on our networks:

### Goerli
`l2geth`: `0.3.0-rc4`
`dtl`: `0.3.0-rc4`
`bs`: `0.3.0-rc4`

### Kovan
`l2geth`: `0.3.1`
`dtl`: `0.3.0`
`bs`: `0.3.2`

### Mainnet
`l2geth`: `0.3.4`
`dtl`: `0.3.0`
`bs`: `0.3.2`

Since mainnet is now up to date to those versions, we should update `master` on this repo.

### CODE REVIEW

File: packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol [hunk 1]
```diff
@@ -100,7 +100,6 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             "Fee was not transferred to relayer."
         );
 
-        // Contract creations are signalled by sending a transaction to the zero address.
```
Comment (snario) [inline packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:103]: This is the only code changed to the contracts Solidity source code from `master` so far. However, the `contracts` package was bumped from `0.3.0` to `0.3.2`.

File: l2geth/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eth-optimism/l2geth",
-  "version": "0.3.1",
+  "version": "0.3.4",
```
Comment (snario) [inline l2geth/package.json:3]: `l2geth` bump ^

File: packages/batch-submitter/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eth-optimism/batch-submitter",
-  "version": "0.3.0",
+  "version": "0.3.2",
```
Comment (snario) [inline packages/batch-submitter/package.json:3]: `batch-submitter` bump ^

File: packages/data-transport-layer/package.json [hunk 4]
```diff
@@ -1,12 +1,14 @@
 {
   "name": "@eth-optimism/data-transport-layer",
-  "version": "0.3.0",
+  "version": "0.3.2",
```
Comment (snario) [inline packages/data-transport-layer/package.json:3]: `dtl` bump ^

**Review by gakonst — APPROVED** [review]
LGTM. Don't forget the rebase!

File: packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol (hunk 1 above)
Comment (gakonst) [reply packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:103]: There was an update to the state dump scripts

**Pull request merged**